src/libraries/Common/tests/System/TestReflectionILAssembly/TestReflectionILAssembly.ilproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.IL">
```

Comment:
Are these new test assemblies really needed, if their content was in one of the existing test assemblies could the tests using them still be made to work?
Each new assembly adds a little build time on everyone’s machine.